floor_generator/test/processor/entity_processor_test.dart

```diff
@@ -120,6 +120,56 @@ void main() {
     expect(actual.fields.length, equals(2));
   });
 
+  test('Ignore getter', () async {
+    final classElement = await _createClassElement('''
+      @entity
+      class Person {
+        @primaryKey
+        final int id;
+      
+        final String name;
+      
+        String get label => "" + id + ": " + name
```

Comment:
💡Dart's string interpolation is pretty powerful. You could give it a try.

```suggestion
        String get label => '$id: $name';
```

Reply:
In this case it is *too* good, because it tries to get the values from outside the class definition:
```
  Failed to load "test/processor/entity_processor_test.dart":
  Unable to spawn isolate: test/processor/entity_processor_test.dart:132:31: Error: Getter not found: 'id'.
          String get label => '$id: $name'
                                ^^
  test/processor/entity_processor_test.dart:132:36: Error: Getter not found: 'name'.
          String get label => '$id: $name'
                                     ^^^^

```
I escaped the dollar signs now.